packages/legend-data-cube/src/stores/core/__tests__/DataCubeQueryFilterSnapshotBuilder.repl-test.ts

```diff
@@ -0,0 +1,168 @@
+/**
```

Comment:
please rename this test to use the new extension `data-cube-test.ts`

Reply:
done